modules/dists/DSIUtil.chpl

```diff
@@ -570,7 +570,14 @@ proc bulkCommTranslateDomain(srcSlice : domain, srcDom : domain, targetDom : dom
   // {1..20 by 4} in {1..20} to {101..120} = {101..120 by 4}
   param needsStridable = targetDom.stridable || srcSlice.stridable;
   var rngs : targetDom.rank*range(targetDom.idxType, stridable=needsStridable);
-  rngs = targetDom.dims();
+  // the following is a hack because split-init treats the following
+  // assignment as an initialization, but that initialization fails on
+  // master (which is why it was split into a separate statement to
+  // begin with).  Introducing the conditional causes the compiler not
+  // to consider it a split initialization, returning to the previous
+  // behavior.
```

Comment:
Issue number?

Also I would probably write this for the workaround:
```
var rngs : targetDom.rank*range(targetDom.idxType, stridable=needsStridable);
rngs; // disable split init
rngs = targetDom.dims();
```

At least this is less complex than the conditional, if you have to comment it anyway.

Reply:
Happily, this seems to have resolved since I opened this PR, so the workaround used in my PR (and on master) can be removed.  I'll close the related issue once I verify some more.